umodbus/server.py

```diff
@@ -12,26 +13,20 @@
 from umodbus.exceptions import ModbusError, ServerDeviceFailureError
 
 
-def get_server(host, port):
-    """ Return :class:`modbus.Server` bound to given port and listening for
-    given host.  :class:`modbus.RequestHandler` is passed as
-    RequestHandlerClass.
+def get_server(server_class, server_address, request_handler_class):
+    """ Return instance of :param:`server_class` with :param:`request_handler`
+    bound to it.
 
-        >>> server = get_server('localhost', 502)
-        >>> server serve_forever()
-
-    :param host: Hostname.
-    :param port: Port number.
-    :return: Server, subclass of socketserver.TCPServer.
-    """
-    return Server((host, port), RequestHandler)
+    This method also binds a :func:`route` method to the server instance.
 
+        >>> server = get_server(TcpServer, RequestHandler, ('localhost', 502))
```

Comment:
The order of the given arguments in this example does not comply with the signature.